PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -0,0 +1,67 @@
+import FWCore.ParameterSet.Config as cms
+
+MicroEventContent = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'drop *',
+        'keep *_slimmedPhotons*_*_*',
+        'keep *_slimmedElectrons_*_*',
+        'keep *_slimmedMuons*_*_*',
+        'keep *_slimmedTaus*_*_*',
+        'keep *_slimmedJets_*_*',
+        'keep *_slimmedJetsAK8_*_*',
+        'keep *_slimmedJetsPuppi_*_*',
+        'keep *_slimmedMETs*_*_*',
+        'keep *_slimmedSecondaryVertices*_*_*',
+        'keep *_cmsTopTaggerMap_*_*',
+        #'keep *_slimmedJetsAK8PFCHSSoftDropSubjets_*_*',
+        #'keep *_slimmedJetsCMSTopTagCHSSubjets_*_*',
+        'keep *_slimmedJetsAK8PFCHSSoftDropPacked_SubJets_*',
+        'keep *_slimmedJetsCMSTopTagCHSPacked_SubJets_*',
+        #'keep *_packedPatJetsAK8_*_*',
+        ## add extra METs
+
+        'keep recoPhotonCores_reducedEgamma_*_*',
+        'keep recoGsfElectronCores_reducedEgamma_*_*',
+        'keep recoConversions_reducedEgamma_*_*',
+        'keep recoSuperClusters_reducedEgamma_*_*',
+        'keep recoCaloClusters_reducedEgamma_*_*',
+        'keep EcalRecHitsSorted_reducedEgamma_*_*',
+
+
+        'drop *_*_caloTowers_*',
+        'drop *_*_pfCandidates_*',
+        'drop *_*_genJets_*',
+
+        'keep *_offlineBeamSpot_*_*',
+        'keep *_offlineSlimmedPrimaryVertices_*_*',
+        'keep patPackedCandidates_packedPFCandidates_*_*',
+
+        'keep double_fixedGridRho*__*',
+
+        'keep *_selectedPatTrigger_*_*',
+        'keep patPackedTriggerPrescales_patTrigger__*',
+        'keep *_l1extraParticles_*_*',
+        'keep L1GlobalTriggerReadoutRecord_gtDigis_*_*',
+        'keep *_TriggerResults_*_HLT',
+        'keep *_TriggerResults_*_PAT', # for MET filters
```

Comment:
... this is a comment beyond this PR though ... this file is a copy of what was in the slimmig_cff